File: mu_wang/9_13/Search Insert Position.py
class Solution(object):
    def searchInsert(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
        index = None
        a = 0
        for i in range(len(nums)):
            if target == nums[i]:
                index = i
                break
            else:
                if target<nums[i]:
                    index= i
                    break
        if index is None:
            index = len(nums)
        return index

File: ziteng/Search Insert Position.java
public class Solution {
    public int searchInsert(int[] nums, int target) {
        int res = 0;
        if (nums.length == 0) return 0;
        for (int i = 0; i < nums.length; i++) {
            if (target <= nums[i]) {
                return i;
            } else {
                
            }
        }
        return nums.length;
    }
}